joschahenningsen/TUM-Live
# Subtitles: Admin

### Motivation and Context
Admins should be able to generate subtitles.


### Description
- Link Worker and Voice Service
- Implement Receiver 
- Add a button for subtitle generation on the admin course page 

### Steps for Testing

- course Admin
- voice-service running @ localhost:5055 (or configured otherwise)
- update VOD file to a local file in database 
    - e.g. ```update tumlive.files set path = '/Users/hansi/Desktop/video.mp4' where stream_id = 11111;```

1. Log in
2. Head to admin course page
3. Generate subtitles (Next to VOD download button)
4. Check database if subtitles have been inserted
5. 🌟
